model-optimizer/mo/back/offline_transformations.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+
+if __name__ == "__main__":
+    try:
+        from openvino.inference_engine import IECore # pylint: disable=import-error
+        from openvino.offline_transformations import ApplyMOCTransformations, CheckAPI # pylint: disable=import-error
+    except Exception as e:
+        print("[ WARNING ] {}".format(e))
```

Comment:
Why you don't use here log.warning (as in other places of mo)? And in other places you use print for warnings, debug info etc. I think it is better to use log.warning, log.debug etc to have the same style in all MO messages.

Reply:
I've started with log.warnig but they actually don't print warnings... If you know the reason please let me know.

Reply:
log.error("message", extra={'is_warning': True}) does work